AIOnboardingARMTemplates/existingClusterOnboarding.json

```diff
@@ -0,0 +1,89 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "aksResourceId": {
+            "type": "string",
+            "metadata": {
+                "description": "AKS Cluster Resource ID"
+            }
+        },
+        "aksResourceLocation": {
+            "type": "string",
+            "metadata": {
+              "description": "Location of the AKS resource e.g. \"East US\""
```

Comment:
Is this the format we are following? Not "eastus"?

Reply:
yes makes sense - my bad. Let me make this update

Reply:
@jatakiajanvi12 let's make sure it's in line with what ARM typically uses. In my experience, templates usually follow the "eastus" format, but my experience is limited.

Reply:
yes you are right. Our current ARM templates also follow that and I tested with "eastus" only but there was a description error